common/src/main/java/com/redhat/cloud/notifications/models/EventAggregationCriterion.java

```diff
@@ -28,13 +51,11 @@ public UUID getApplicationId() {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
-            return true;
+        if (o == null || getClass() != o.getClass()) {
+            return false;
         }
-        EmailAggregationKey that = (EmailAggregationKey) o;
-        return Objects.equals(getOrgId(), that.getOrgId()) &&
-            Objects.equals(getBundle(), that.getBundle()) &&
-            Objects.equals(getApplication(), that.getApplication());
+        EventAggregationCriterion that = (EventAggregationCriterion) o;
+        return Objects.equals(bundleId, that.bundleId) && Objects.equals(applicationId, that.applicationId) && Objects.equals(orgId, that.orgId) && Objects.equals(bundle, that.bundle) && Objects.equals(application, that.application);
```

Comment:
Please make the `hashCode` method consistent with `equals`: `applicationId` and `bundleId` need to be included.